main/guides/getting-started/README.md

```diff
@@ -159,6 +160,24 @@ For more examples and ideas, visit:
 
 </details>
 
+::: tip Using a Mac?
```

Comment:
Maybe include "troubleshooting" in the tip line?

Also, my habit is to take error messages and put them into search boxes / search engines, and I think I'm not alone; so let's put the text of these diagnostics in the body of this box so that folks who search for them will find this box:

 - `Segmentation fault` or `start_agd.sh: line 10:   134 Segmentation fault` from https://github.com/Agoric/agoric-3-proposals/issues/30
 - `signature verification failed` from https://github.com/Agoric/dapp-offer-up/issues/10